app/src/main/java/ch/epfl/culturequest/ui/SearchActivity.java

```diff
@@ -53,7 +53,7 @@ public class SearchActivity extends AppCompatActivity {
     private Button searchUsers, searchCities;
     private EditText search;
     private MutableLiveData<Boolean> searchingForUsers = new MutableLiveData<>(true);
-    public static final int NUMBER_USERS_TO_DISPLAY = 5, NUMBER_CITIES_TO_DISPLAY = 10;
+    public static final int NUMBER_USERS_TO_DISPLAY = 5, NUMBER_CITIES_TO_DISPLAY = 8;
```

Comment:
Did you change this because of performance issues?

Reply:
not for cities in europe it doesnt change anything, but maybe later if we decide to have all the cities in the world, it might make performance better